types/index.d.ts

```diff
@@ -2566,6 +2566,28 @@ declare namespace math {
      */
     setUnion<T extends MathCollection>(a1: T, a2: MathCollection): T
 
+    /*************************************************************************
+     * Signal functions
+     ************************************************************************/
+    /**
+     * Compute the transfer function of a zero-pole-gain model.
+     * @param z Zeroes of the model
+     * @param p Poles of the model
+     * @param k Gain of the model
+     * @returns The transfer function as array of numerator and denominator
+     */
+    zpk2tf<T extends MathCollection>(z: T, p: T, k?: number): T
```

Comment:
The return type of the definition needs to be `{ w: T, h: T }` instead of `T`, right?

Reply:
right

Reply:
Ah, yes, I see 👍